packages/pluggableWidgets/rich-text-web/src/components/__tests__/__snapshots__/RichText.spec.tsx.snap

```diff
@@ -2,16 +2,21 @@
 
 exports[`RichText render DOM structure 1`] = `
 <div
-  className="widget-rich-text"
+  className="widget-rich-text editor-text"
   style={
     Object {
-      "height": "auto",
-      "width": "100%",
+      "height": "320px",
```

Comment:
what caused this change on height and width? ~~couldn't find the configuration~~
Never mind, I found where it comes from, but want to know is this done on purpose?
https://github.com/mendix/widgets-resources/pull/1305/files#diff-9dcae931701667f45bdf013a1875c434c0ca35454b4cda02ea88aa91dff722e7R29-R30